demos/crossroad_camera_demo/README.md

```diff
@@ -1,5 +1,7 @@
 # Crossroad Camera C++ Demo
 
+![](./crossroad_camera.gif)
```

Comment:
The usage is valid, but the source video doesn't seem to be representative enough.
`security_barrier_camera.gif` is also run on a bit different video: the demo name suggests that the camera is placed at level of barrier, but not above of passing cars.

@vladimir-dudnik should the gifs be kept?

Reply:
I think we can use a few subsequent frames from internal validation set, just resize it to lower resolution.
@yssemaev hope there is no legal issues with this, as it should not fall into dataset redistribution case. It is just screenshots of demo execution.

Reply:
Hi, I have no objections if the dataset(s) was approved for Publishing and Distribution cases.